tools/mo/openvino/tools/mo/back/offline_transformations.py

```diff
@@ -83,6 +86,8 @@ def read_model(path_to_xml):
     if "compress_fp16" in argv and argv.compress_fp16:
         compress_model(func)
 
+    apply_fused_names_cleanup(func)
```

Comment:
I hope POT also calls this part before IR serialization. Correct?

Reply:
@slyalin POT calls `apply_pot_transformations` for GNA only. It is another place.

Reply:
POT should also call this pass no matter if they are using ngraph transformations or not. POT could've read the IR with fused names.

Reply:
> POT should also call this pass no matter if they are using ngraph transformations or not. POT could've read the IR with fused names.

I suppose there are 2 cases: when POT reads the IR with/without fused_names and when it serializes model by himself

For both cases we should have dedicated tickets

Reply:
cc @AlexKoff88

Reply:
I got what happens here but don't know what is really being solved. Can anybody explain it briefly? cc'ed @l-bat, @KodiaqQ .

Reply:
As I understand it relates to `rt_info`, so I think it's not a problem for POT to call this pass when POT serializes model or reads model with/without fused_names.

Reply:
> I got what happens here but don't know what is really being solved. Can anybody explain it briefly? cc'ed @l-bat, @KodiaqQ .

Brief description is available in epic 91551